src/stores/ImportStore.ts

```diff
@@ -53,5 +53,5 @@ export default class ImportStore {
   }
 
   @action
-  public cancelImport = () => this.app.routerStore.goBack()
+  public cancelImport = () => this.app.routerStore.push('/create-wallet')
```

Comment:
whats wrong with using goBack? I think push will work fine,  but it makes the history object stack longer

Reply:
reverted back to using goBack